Makefile.am

```diff
@@ -5,8 +5,8 @@
 ACLOCAL_AMFLAGS = -I m4
 
 # Make targets will be run in each subdirectory. Order is significant.
-SUBDIRS = include platform constants tridiagonal mpp memutils fms	\
-mosaic time_manager diag_manager drifters axis_utils horiz_interp	\
+SUBDIRS = include sort platform constants tridiagonal mpp memutils	\
```

Comment:
Note here that a new directory "sort" has been added. Order is significant, it must be before mpp.